trouble with Version 0.9.5

Today I updated to version 0.9.5. All of the commands are not working anymore. Also build and debug symbols at the bottom of vs code lost. My vs code version is 1.12.2

Same problem with 9.6 too
I am working on Linux Mint (Ubuntu 5.4.0-6ubuntu1~16.04.4) 
I am not sure if I understand you correct but this is what I get when open developer tool's console

```
[Extension Host] Error from cmake-server process: CMake Error: cmake version 3.5.1
Usage: cmake -E <command> [arguments...]
Available commands: 
  chdir dir cmd [args...]   - run command in a given directory
  compare_files file1 file2 - check if file1 is same as file2
  copy <file>... destination  - copy files to destination (either file or directory)
  copy_directory <dir>... destination   - copy content of <dir>... directories to 'destination' directory
  copy_if_different <file>... destination  - copy files if it has changed
  echo [<string>...]        - displays arguments as text
  echo_append [<string>...] - displays arguments as text but no new line
  env [--unset=NAME]... [NAME=VALUE]... COMMAND [ARG]...
                            - run command in a modified environment
  environment               - display the current environment
  make_directory <dir>...   - create parent and <dir> directories
  md5sum <file>...          - compute md5sum of files
  remove [-f] <file>...     - remove the file(s), use -f to force it
  remove_directory dir      - remove a directory and its contents
  rename oldname newname    - rename a file or directory (on one volume)
  tar [cxt][vf][zjJ] file.tar [file/dir1 file/dir2 ...]
                            - create or extract a tar or zip archive
  sleep <number>...         - sleep for given number of seconds
  time command [args...]    - run command and return elapsed time
  touch file                - touch a file.
  touch_nocreate file       - touch a file but do not create it.
Available on UNIX only:
  create_symlink old new    - create a symbolic link new -> old

```

`extensionHost.ts:285 [Extension Host] [cmake output]: cmake version 3.0.2
extensionHost.ts:285 [Extension Host] [cmake output]: 
extensionHost.ts:285 [Extension Host] [cmake output]: CMake suite maintained and supported by Kitware (kitware.com/cmake).
extensionHost.ts:285 [Extension Host] cmake exited with return code 0
extensionHost.ts:285 [Extension Host] Loading CMake Tools from /Path/To/My/Application/.vscode/.cmaketools.json
extensionHost.ts:285 [Extension Host] Started new CMake Server instance with PID 2104
extensionHost.ts:285 [Extension Host] Error from cmake-server process: CMake Error: cmake version 3.0.2
Usage: cmake -E [command] [arguments ...]
Available commands: 
  chdir dir cmd [args]...   - run command in a given directory
  compare_files file1 file2 - check if file1 is same as file2
  copy file destination     - copy file to destination (either file or directory)
  copy_directory source destination   - copy directory 'source' content to directory 'destination'
  copy_if_different in-file out-file  - copy file if input has changed
  echo [string]...          - displays arguments as text
  echo_append [string]...   - displays arguments as text but no new line
  environment               - display the current environment
  make_directory dir        - create a directory
  md5sum file1 [...]        - compute md5sum of files
  remove [-f] file1 file2 ... - remove the file(s), use -f to force it
  remove_directory dir      - remove a directory and its contents
  rename oldname newname    - rename a file or directory (on one volume)
  tar [cxt][vfz][cvfj] file.tar [file/dir1 file/dir2 ...]
                            - create or extract a tar or zip archive
  sleep <number>...         - sleep for given number of seconds
  time command [args] ...   - run command and return elapsed time
  touch file                - touch a file.
  touch_nocreate file       - touch a file but do not create it.
Available on UNIX only:
  create_symlink old new    - create a symbolic link new -> old

d.logExtensionHostMessage @ extensionHost.ts:285
extensionHost.ts:285 [Extension Host] Error from cmake-server process: 
d.logExtensionHostMessage @ extensionHost.ts:285
extensionHost.ts:285 [Extension Host] (node:2002) DeprecationWarning: os.tmpDir() is deprecated. Use os.tmpdir() instead.
d.logExtensionHostMessage @ extensionHost.ts:285`

--- a/src/util.ts
+++ b/src/util.ts
@@ -189,12 +189,12 @@ export interface Version {
   patch: number;
 }
 export function parseVersion(str: string): Version {
-  const version_re = /(\d+)\.(\d+).(\d+)/;
+  const version_re = /(\d+)\.(\d+)\.(\d+)/;
   const mat = version_re.exec(str);
   if (!mat) {
     throw new Error(`Invalid version string ${str}`);
   }
-  const [major, minor, patch] = mat!;
+  const [, major, minor, patch] = mat;
   return {
     major: parseInt(major),
     minor: parseInt(minor),
@@ -206,10 +206,18 @@ export function versionGreater(lhs: Version, rhs: Version|string): boolean {
   if (typeof(rhs) === 'string') {
     return versionGreater(lhs, parseVersion(rhs));
   }
-  return lhs.major > rhs.major ||
-      (lhs.major == rhs.major && lhs.minor > rhs.minor) ||
-      (lhs.major == rhs.major && lhs.minor == rhs.major &&
-       lhs.patch == lhs.patch);
+  if (lhs.major > rhs.major) {
+    return true;
+  }
+  else if (lhs.major === rhs.major) {
+    if (lhs.minor > rhs.minor) {
+      return true;
+    }
+    else if (lhs.minor === rhs.minor) {
+      return lhs.patch > rhs.patch;
+    }
+  }
+  return false;
 }
 
 export function versionEquals(lhs: Version, rhs: Version|string): boolean {

--- a/test/extension.test.ts
+++ b/test/extension.test.ts
@@ -345,6 +345,12 @@ suite("Utility tests", () => {
         assert.strictEqual(info.compileFlags[0], '/Z+:some-compile-flag');
         assert.strictEqual(info.compiler, 'cl.exe');
     });
+    test('Version compare', async function () {
+        assert(!util.versionGreater(util.parseVersion('3.0.2'), '3.7.1'));
+        assert(util.versionGreater(util.parseVersion('1.0.1'), '1.0.0'));
+        assert(util.versionGreater(util.parseVersion('1.1.2'), '1.0.3'));
+        assert(util.versionGreater(util.parseVersion('1.2.3'), '0.4.5'));
+    });
     test('Can access the extension API', async function () {
         this.timeout(40000);
         const api = await getExtension();

